### PR TITLE
356 support multiline abstract

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

#356 

## Purpose/Implementation Notes

- adds `ProjectAbstractDetail`
  - supports `\n` as a new line character to break up abstract into multiple lines
- renames file `ProjectPublicationsDetails` -> `ProjectPublicationsDetail`

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

N/A tested locally, with existing data and faked data we expect to see

## Checklist

- [X] Lint and unit tests pass locally with my changes

## Screenshots

n/a

### CODE REVIEW

**Review by nozomione — APPROVED** [review]
LGTM
(I added a minor suggestion, but it's up to you!)

File: client/src/components/ProjectAbstractDetail.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+import { Paragraph } from 'grommet'
+
+export const ProjectAbstractDetail = ({ abstract }) =>
+  abstract.split('\n').map((line) => <Paragraph>{line}</Paragraph>)
```
[the remark this answers is not in the document]
Comment (davidsmejia) [reply client/src/components/ProjectAbstractDetail.js:5]: yeah I was thinking about that, if we end up touching this part of it again I think that would be worthwhile, however probably not worth it until the need arises

[the remark this answers is not in the document]
Comment (nozomione) [reply client/src/components/ProjectAbstractDetail.js:5]: Got it!

**Comment by nozomione** [comment]
@davidsmejia 
A quick note: Currently a paragraph doesn't have any margin in the abstract(which makes sense for a single parahraph), and perhaps add a top margin for multi-paragraph for better readability? Thank you, David!

e.g.) Using indices or `sytled-components` 
```javascript
abstract.split('\n').map((line, i) => (
    <Paragraph margin={{ top: i ? 'small' : 'none' }}>{line}</Paragraph>
``` 


(Search results)

image

(Project details)

image

e.g., ) After adding a `small` margin:

(Search results)

image

(Project details)

image

**Pull request merged**